posts/rails_memory.md

```diff
@@ -119,12 +139,16 @@ CPU: 0.03% -> 1.9% -> 0.05%
 Memory: 30% -> 39% -> 39%
 ```
 
+<!-- part 9 -->
+
 All my requests were handled but the memory did not decrease.
 It seemed like the Puma were not the root of the memory problem.
 I had to dig deeper and maybe it was ruby.
```

Comment:
- I had to dig deeper and maybe it was ruby  -> I had to dig deeper, maybe the real culprit had been Ruby all along

Reply:
https://github.com/HoangNguyen679/aphelios/pull/1/commits/86d38bb661a525913e5ef5fbef28102c0c25fa93 fixed